cartographer_ros/cartographer_ros/node_options.cc

```diff
@@ -40,6 +40,14 @@ NodeOptions CreateNodeOptions(
       lua_parameter_dictionary->GetDouble("pose_publish_period_sec");
   options.trajectory_publish_period_sec =
       lua_parameter_dictionary->GetDouble("trajectory_publish_period_sec");
+  if (lua_parameter_dictionary->HasKey("publish_to_tf")) {
+    options.publish_to_tf =
+        lua_parameter_dictionary->GetBool("publish_to_tf");
+  }
+  if (lua_parameter_dictionary->HasKey("publish_tracked_pose_msg")) {
+    options.publish_tracked_pose_msg =
+        lua_parameter_dictionary->GetBool("publish_tracked_pose_msg");
```

Comment:
I would remove `_msg` from the option name, it doesn't provide additional value.

Reply:
done